MVC/controllers.py

```diff
@@ -1,41 +1,82 @@
-from MVC.models import Yard, Bed, Plant
+from typing import List
+from typing import Optional
+
+from MVC.models import Bed
+from MVC.models import BedRequestCreate
+from MVC.models import Plant
+from MVC.models import PlantRequestCreate
+from MVC.models import Yard
+from MVC.models import YardRequestCreate
+from utils.db import session
+
 
 class ControllerYards:
-    def __init__(self):
-        pass
+    _yards: List[Yard] = []
 
-        
-    def get_all(self) -> dict:
-        yards = self.fetch_all()
-        return ( { "message" : "Listing all yards", "yards" : yards } )
+    def __init__(self):
+        self._yards = self.get()
 
+    def get(self, request: Optional[List[int]] = None) -> List[Yard]:
+        self._yards = []
+        if (request):
+            for _ in request:
+                yard = session.query(Yard).get(_)
+                if (yard):
```

Comment:
en python no queremos poner mucho comentario pero si un 
# this method allows get one, get multi and get all 

Aclara todo, vale la pena ponerlo